pkgs/development/python-modules/fontfeatures/default.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, fetchpatch
+, beziers
+, fonttools
+, fs
+, glyphtools
+, lxml
+, pytestCheckHook
+, youseedee
+}:
+
+buildPythonPackage rec {
+  pname = "fontfeatures";
+  version = "1.7.4";
+
+  src = fetchPypi {
+    pname = "fontFeatures";
+    inherit version;
+    hash = "sha256-t5VHwDVQupBTqemFhkhwCXTSzeAKA0gIjWb7BSjTcBw=";
+  };
+
+  patches = [
+    # https://github.com/simoncozens/fontFeatures/pull/60
+    (fetchpatch {
+      url = "https://github.com/simoncozens/fontFeatures/pull/60/commits/c4ac688b724d765b30ffdc3d964e40935feb7264.patch";
+      sha256 = "sha256-vtDeYkI0R/ZwUBIkqnLNJ0lMwE45OXrbSIS1Fv+hsfo=";
+    })
+  ];
+
+  propagatedBuildInputs = [
+    beziers
+    fonttools
+    fs
+    glyphtools
+    lxml
+    youseedee
+  ];
+  nativeCheckInputs = [
+    pytestCheckHook
+  ];
+
+  disabledTestPaths = [
+    # These tests require babelfont but we have to leave it out and skip them
+    # to break the cyclic dependency with babelfont.
+    "tests/test_shaping_generic.py"
+    "tests/test_shaping_harfbuzz.py"
+  ];
+
+  pytestFlagsArray = [
+    # Failing upstream:
+    # https://github.com/simoncozens/fontFeatures/pull/60#issuecomment-1518963962
+    # Have to write out the -k option instead of using disabledTests because
+    # of the test parameter.
+    "-k 'not (test_round_trip and mark_attachment)'"
```

Comment:
Is it possible to use `disabledTests` here instead?

Reply:
No, that didn't work because it's a parametrized test... but the test is passing in the latest upstream release now.